core/trino-main/src/main/java/io/trino/eventlistener/EventListenerConfig.java

```diff
@@ -26,6 +27,7 @@
 public class EventListenerConfig
 {
     private List<File> eventListenerFiles = ImmutableList.of();
+    private int maxConcurrentQueryCompletedEvents = 100;
```

Comment:
Why 100? IMO, the default should be unlimited. The specific number would depend on which event listeners you're using and whether they can handle a given number of concurrent calls.

Reply:
@martint https://github.com/trinodb/trino/pull/22994